src/components/BookmarkWarning/BookmarkWarning.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import Icon from "../Icon/Icon";
+import { ButtonX, StyledBookmarkWarning, Text } from "./BookmarkWarning.styles";
+
+export type BookmarkWarningProps = {
+  /**
+   * if hidden, return null so that nothing gets rendered
+   */
+  hidden?: boolean;
+} & React.ButtonHTMLAttributes<HTMLButtonElement>;
```

Comment:
BookmarkWarning should not be a button, so it shouldn't inherit HTMLButtonElement props.

Reply:
Ok, fixed